### PR TITLE
Remove PropertiesChanged list from Payload classes

### PR DESCRIPTION
**Pull request by gortegaMS**
# Type of PR

- [ ] Documentation changes
- [x] Code changes
- [x] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## PR Checklist

Remove PropertiesChanged list from Payload classes

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] My code follows the code style of this project.
- [x] I ran lint checks locally prior to submission.
- [x] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Does this introduce a breaking change

- [ ] YES
- [x] NO

## Validation

- [x] Unit tests updated and ran successfully
- [ ] Update documentation or issue referenced above

## Issues Closed or Referenced

- Closes https://github.com/retaildevcrews/wcnp/issues/624

### CODE REVIEW

File: src/LodeRunner.Core/Extensions/LoadTestConfigExtensions.cs [hunk 1]
```diff
@@ -20,32 +20,27 @@ public static class LoadTestConfigExtensions
         /// If payloadPropertiesChanged list exists, only the listed properties are converted.
         /// </summary>
```
Comment (aflinchb) [inline src/LodeRunner.Core/Extensions/LoadTestConfigExtensions.cs:21]: nit: remove reference to payloadProperteisChanged list in summary comment :)

File: src/LodeRunner.Core/Models/TestRunPayload.cs [hunk 2]
```diff
@@ -50,7 +42,7 @@ public class TestRunPayload : BasePayload
         /// <value>
         /// The CreatedTime.
         /// </value>
-        public DateTime CreatedTime { get => this.testRun.CreatedTime; set => this.SetField(this.testRun, value); }
+        public DateTime? CreatedTime { get; set; }
```
Comment (aflinchb) [inline src/LodeRunner.Core/Models/TestRunPayload.cs:45]: I noticed that CreatedTime and StartTime have the required attribute in the TestRun model class.  Does a default value for these properties get set in when the TestRun object is created from the payload?

**Review by aflinchb — APPROVED** [review]
Approved with a comment/question :)

**Review by alfredoihernandez — APPROVED** [review]

File: src/LodeRunner.Core/Extensions/LoadTestConfigExtensions.cs (hunk 1 above)
Comment (gortegaMS) [reply src/LodeRunner.Core/Extensions/LoadTestConfigExtensions.cs:21]: good eye., removed

File: src/LodeRunner.Core/Models/TestRunPayload.cs (hunk 2 above)
Comment (gortegaMS) [reply src/LodeRunner.Core/Models/TestRunPayload.cs:45]: Good question, we do have Automapper for that and on this PR I also refactored the Extension method "IgnoreNullFields" to check for Field.Value not null instead PropertiesChangedList to do the mapping when creating TestRuns from TestRunPayload

**Pull request merged**